src/main/java/online/book/store/mapper/CategoryMapper.java

```diff
@@ -0,0 +1,17 @@
+package online.book.store.mapper;
+
+import online.book.store.config.MapperConfig;
+import online.book.store.dto.request.category.CategoryRequestDto;
+import online.book.store.dto.response.category.CategoryResponseDto;
+import online.book.store.model.Category;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+
+@Mapper(config = MapperConfig.class)
+public interface CategoryMapper {
+    CategoryResponseDto toDto(Category category);
+
+    @Mapping(target = "id", ignore = true)
+    @Mapping(target = "deleted", ignore = true)
```

Comment:
I have none of this in my CategoryMapper and it works.

Reply:
I had a warn message in the console, and after adding these, the message was gone, but now, when I deleted it, that message don't back, weird

Reply:
Then please remove unnecessary annotations